nx.json

```diff
@@ -5,7 +5,7 @@
     "default": {
       "runner": "nx/tasks-runners/default",
       "options": {
-        "cacheableOperations": ["build", "lint", "test", "e2e"]
+        "cacheableOperations": ["build", "lint", "test", "e2e", "genfiles"]
```

Comment:
is `genfiles` cacheable? usually generated files are not even tracked, or tracked outside of the module they are generated from/to, which would prevent the caching of their opertaions?

Reply:
https://nx.dev/concepts/how-caching-works